### PR TITLE
Make updating instructions more obvious

### PR DESCRIPTION
**Pull request by lindsaymarkward**
Make updating instructions more obvious.
Could I also suggest that the "Update Available" option that I see on my States page links directly to this fragment? Currently it just links to the main page and I have to scroll through to figure out how to update.
Thanks

### CODE REVIEW

**Comment by lindsaymarkward** [comment]
It looks like I had to restart Home Assistant, so I added that to the instructions.

File: source/getting-started/index.markdown [hunk 1]
```diff
@@ -284,11 +284,13 @@ For additional help, in addition to this site, there are four sources:
  - [Gitter Chatroom](https://gitter.im/balloob/home-assistant) for general Home Assistant discussions and questions.
  - [GitHub Page](https://github.com/balloob/home-assistant/issues) for issue reporting.
 
+### Updating
+To update Home Assistant to the latest release run: `pip3 install --upgrade homeassistant`  
+You then have to restart Home Assistant for the changes to take effect. If you have installed it as an autostarting daemon (see below), then run: `sudo service hass-daemon restart`
```
Comment (fabaff) [inline source/getting-started/index.markdown]: I think that we should not add a daemon-specific command here. From my point if a user is using some kind of autostarting mechanism she/he is aware of the commands to start/stop/restart the used daemon or init implementation.

**Comment by fabaff** [comment]
I like the idea of a new section Makes it indeed more obvious.

**Comment by lindsaymarkward** [comment]
Sure. Adjust to whatever is best. I expected it to work when I updated, but it didn't do anything new until I restarted it, so that's something newbies like me would need to know.

**Pull request merged**